recipes/arch/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "arch" %}
```

Comment:
I'm not sure about using this name. Other package managers use this name for another package.

Reply:
Understand your concern. I changed the package name to `arch-py` and left the `name` there to refer to the pypi file path.

```
package:
    name: arch-py
```

I take it that when other users install the package this way, there shouldn't be any conflict.

Reply:
@isuruf, ping me if you still have any lingering concerns :)